minimal/docker-compose.yml

```diff
@@ -53,4 +53,13 @@ services:
     volumes_from:
       - {{ project_name }}_web
     ports:
-      - "0.0.0.0:80:80"
\ No newline at end of file
+      - "0.0.0.0:80:80"
+
+  {{ project_name }}_certbot:
+    restart: always
+    build: ./compose/certbot/production
+    depends_on:
+      - {{ project_name }}_web
+    volumes_from:
```

Comment:
@pslobo you can limit the volumes here like this:
```
    volumes:
      - ./staticfiles:/code/staticfiles
```

I am going to fix the volumes restrictions in the rest of the code when i address this issue https://github.com/psychok7/django-project-template-yadpt/issues/5